src/mame/trs/gime.cpp

```diff
@@ -1329,42 +1329,26 @@ inline uint16_t gime_device::get_lines_per_row()
 	uint16_t lines_per_row;
 	if (m_legacy_video)
 	{
-		switch(m_ff22_value & MODE_AG)
+		if (m_ff22_value & MODE_AG)
 		{
-			case 0:
+			static int gime_legacy_lines_per_row_graphic[8] =
 			{
-				// http://cocogamedev.mxf.yuku.com/topic/4299238#.VyC6ozArI-U
-				static int ff9c_lines_per_row[16] =
-				{
-					11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 4, 3, 2, 1, 12
-				};
-
-				int i = m_gime_registers[0x0C] & 0x0F;
-				//lines_per_row = 12;
-				lines_per_row = ff9c_lines_per_row[i];
-				break;
-			}
+				3, 3, 3, 2, 2, 1, 1, 1
+			};
```

Comment:
This is non-constant static data in a function.  @ajrhacker can you please review code before merging it?